pkg/diskmaker/controllers/lvset/device_age.go

```diff
@@ -0,0 +1,62 @@
+package lvset
+
+import (
+	"sync"
+	"time"
+)
+
+var (
+	// deviceMinAge is the minimum age for a device to be considered safe to claim
+	// otherwise, it could be a device that some other entity has attached and we have not claimed.
+	deviceMinAge = time.Minute
```

Comment:
we can do this in a follow up but IMO this should be configurable via CLI option in diskmaker, so as certain deployments can run with different values potentially without requiring code change. This however may require propagating upto `LocalVolumeSet` object if this has to be configurable. We can do this in a follow up.